### PR TITLE
SidebarRow: hide description line if blank

### LINKED ISSUE #72: Permissions: Handle fully sandboxed apps
If apps are completely sandboxed, the sidebar description line is empty. We should probably have something here

Screenshot from 2020-06-04 11 56 52@2x

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/91835052-permissions-handle-fully-sandboxed-apps?utm_campaign=plugin&utm_content=tracker%2F56794370&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F56794370&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by danrabbit**
Fixes #72

### CODE REVIEW

File: src/Permissions/Widgets/SidebarRow.vala [hunk 1]
```diff
@@ -68,8 +72,14 @@ public class Permissions.SidebarRow : Gtk.ListBoxRow {
             }
         }
 
-        var description = string.joinv (", ", current_permissions.data);
-        description_label.label = "<small>%s</small>".printf (description);
-        set_tooltip_text (description);
+        if (current_permissions.length > 0) {
+            var description = string.joinv (", ", current_permissions.data);
```
Comment (tintou) [inline src/Permissions/Widgets/SidebarRow.vala:76]: I'm sure that it is not i18n friendly as not all language use this specific comma character for this

File: src/Permissions/Widgets/SidebarRow.vala [hunk 2]
```diff
@@ -68,8 +72,15 @@ public class Permissions.SidebarRow : Gtk.ListBoxRow {
             }
         }
 
-        var description = string.joinv (", ", current_permissions.data);
-        description_label.label = "<small>%s</small>".printf (description);
-        set_tooltip_text (description);
+        if (current_permissions.length > 0) {
+            /// Translators: This is a delimeter that separates types of permissions in the sidebar description
```
Comment (jeremypw) [inline src/Permissions/Widgets/SidebarRow.vala:76]: Spelling: "delimiter".

Comment (danrabbit) [reply src/Permissions/Widgets/SidebarRow.vala]: Oops, thanks!

**Comment by jeremypw** [comment]
This plug is crashing in master (as well as the Keyboard plug) - I think after installing the latest updates from AppCenter.  Need to investigate that before reviewing this again.

**Comment by jeremypw** [comment]
It is not obvious to a new user that the Permissions tab only applies to Flatpak apps.  We can guard against no installed Flatpak apps in it but it may be better to hide it or make it insensitive when there are no Flatpak apps installed.

**Pull request merged**